CMakeLists.txt

```diff
@@ -240,7 +240,7 @@ set(
 # latest version of a compiler or platform.
 # See the following for more information:
 #   https://www.boost.org/doc/libs/1_77_0/doc/html/boost_asio/using.html
-list(APPEND IRODS_COMPILE_DEFINITIONS_PRIVATE "BOOST_ASIO_HAS_STD_INVOKE_RESULT")
+#list(APPEND IRODS_COMPILE_DEFINITIONS_PRIVATE "BOOST_ASIO_HAS_STD_INVOKE_RESULT") TODO Perhaps this isn't necessary for 1.81.
```

Comment:
The server now compiles without this.

If all unit tests and core tests pass, then it is safe to remove.

Reply:
Removed